src/utils.cpp

```diff
@@ -190,6 +193,9 @@ TextChat* TextChat::unserialize(class MegaClient *client, string *d)
     ou = MemAccess::get<handle>(ptr);
     ptr += sizeof ou;
 
+    ts = (uint32_t)MemAccess::get<time_t>(ptr);
```

Comment:
I don't understand the types here.
`ts` is `m_time_t`. Shouldn't that type be used instead of `uint32_t` and `time_t` in this line and the line below?